deps/include/jconfigint.h

```diff
@@ -0,0 +1,9 @@
+#ifndef INLINE
+#if defined(__GNUC__)
```

Comment:
The jconfigint.h is pretty much what libjpeg-turbo generates on my mac and since it seemed to work for debian+osx+windows I just assumed it's probably pretty good..
At least it worked better (and wider) than the define in the .gyp